mono/metadata/w32file-unix.c

```diff
@@ -4885,6 +4910,12 @@ mono_w32file_init (void)
 	finds = g_hash_table_new_full (g_direct_hash, g_direct_equal, NULL, finds_remove);
 	mono_coop_mutex_init (&finds_mutex);
 
+#if HOST_DARWIN
+	libc_handle = dlopen ("/usr/lib/libc.dylib", 0);
+	g_assert (libc_handle);
+	clonefile_ptr = (clonefile_fn)dlsym (RTLD_DEFAULT, "clonefile");
```

Comment:
We're not using `libc_handle` I don't get why are we dlopen'ing it.

Reply:
Uh, my fault, should be used instead of RTLD_DEFAULT